qt/scientific_interfaces/Indirect/IndirectWorkspaceNames.h

```diff
@@ -0,0 +1,28 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+#include <string>
+#include <utility>
+
+#include "DllConfig.h"
+#include "IndexTypes.h"
+#include "IndirectFitData.h"
+#include "MantidAPI/MatrixWorkspace.h"
```

Comment:
Just wondering if all these includes are needed?

Reply:
They are not good spot